core/src/main/scala/doric/syntax/package.scala

```diff
@@ -0,0 +1,59 @@
+package doric
+
+import cats.implicits._
+import org.apache.spark.sql.{Column, functions => f}
+import org.apache.spark.sql.catalyst.expressions.{ElementAt, Expression, LambdaFunction, UnresolvedNamedLambdaVariable}
+
+package object syntax {
+
+  /**
+    * Abstract method for Array Columns and Map Columns
+    *
+    * Returns element of array at given index in value if column is array. Returns value for
+    * the given key in value if column is map.
+    *
+    * @param dc doric column where the item is to be searched
+    * @param key doric column
+    * @tparam T type of the doric column where the item is to be searched
+    * @tparam K type of "key" doric column to perform the search
+    * @tparam V type of "value" doric column result
+    */
+  @inline def elementAtAbstract[T, K, V](
```

Comment:
Shouldn't be all these elements at least private at the syntax level?

Reply:
I do not get this, they are at the syntax package object

Reply:
done